Source/KSCrash/Recording/Tools/KSLogger.h

```diff
@@ -260,6 +260,14 @@ bool kslog_setLogFilename(const char* filename, bool overwrite);
 /** Clear the log file. */
 bool kslog_clearLogFile();
 
+/** Set whether or not to log to stdout.
+ *
+ * @param enabled Whether or not stdout logging should be enabled.
+ *
+ * By default stdout logging is enabled.
+ */
+void kslog_setLogToStdout(bool enabled);
```

Comment:
Both of those are still needed.

KSCrash already stores a fresh logfile to disk on every run. When the app crashes, `kscrash_setAddConsoleLogToReport` tells KSCrash to read the logfile and add it line-by-line as an array of strings to the JSON crash report that's being written.

`kscrash_setPrintPreviousLog` tells KSCrash to look for a logfile hanging around from the last crash, and if it finds it, print it to the console. It's mostly to aid during debugging crash report generation so that you can quickly see how far the reporter got before something died.